libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Recognizers/Microsoft.AdaptiveCardRecognizer.schema

```diff
@@ -1,15 +0,0 @@
-{
-    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
-    "$role": "implements(Microsoft.IRecognizer)",
-    "title": "Cross-trained Recognizer Set",
-    "description": "Recognizer for detecting the value response from an Adaptive Card.",
-    "type": "object",
```

Comment:
Was this removed in the code?

Reply:
YEah schema file with no backing class.

---
In reply to: [410954399](https://github.com/microsoft/botbuilder-dotnet/pull/3758#discussion_r410954399) [](ancestors = 410954399)